package-lock.json

```diff
@@ -4078,7 +4078,25 @@
 				"eslint-plugin-jsx-a11y": "^6.2.1",
 				"eslint-plugin-react": "^7.12.4",
 				"eslint-plugin-react-hooks": "^1.6.0",
+				"globals": "^12.0.0",
```

Comment:
This change shouldn't be part of this PR. I don't see this dependency added to `package.json` file.